BudgetSquirrel.Api/ClientApp/src/app/splash-page/splash-page.component.html

```diff
@@ -0,0 +1,30 @@
+<div class="splash">
+  <div class="vert-section">
+    <div class="vert-section__content splash__summary-section">
+      <div class="vert-section__column splash__section__content--grow z-6">
+        <h1 class="color-txt__primary-dark">
+            Budget Squirrel is a budget management tool
+            designed for both control freeks and the
+            laid back.
+        </h1>
+        <p class="color-txt__primary-dark splash__section__content--no-grow">
+            A budget management tool designed for both
+            control freeks and the laid back.
+        </p>
+        <a class="register-btn button button--primary button--wide" routerLink="{{routes.SIGN_IN}}">Sign Up</a>
+        <p class="signin-label color-txt__primary-dark">Already have an account?</p>
+        <a class="color-txt__accent" routerLink="{{routes.SIGN_IN}}">Login</a>
```

Comment:
this `routerLink="{{routes.SIGN_IN}}` is fine and no need to change but you can also do `[routerLink]="routes.SIGN_IN"` Just as heads up I don't think there is a right or wrong way just preferences.